profileGenerator.js

```diff
@@ -0,0 +1,31 @@
+import gridGenerator from './gridGenerator.js';
+
+const profileGenerator = (object) => {
+  // Element creation
+  const card = document.createElement('div');
+  card.classList.add('speakers__card');
+  const imagesDiv = document.createElement('div');
+  imagesDiv.classList.add('speakers__card__images');
+  const textDiv = document.createElement('div');
+  textDiv.classList.add('speakers__card__text');
+  const img = document.createElement('img');
```

Comment:
Please add alt attribute to each img element on your web pages. It is crucial for accessibility, especially for the people who has visual impairments.